VTEX - Audience API.json

```diff
@@ -0,0 +1,320 @@
+                                "example": {
+                                    "error": "Invalid input or account not detected."
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/api/price-table-mapper/pvt/mapping/{audienceId}": {
+            "get": {
+                "tags": [
+                    "Price Table Mapper"
+                ],
+                "summary": "Get price table mapping",
+                "description": "Fetches the price table mapping for a specific audience.\r\n\r\n## Permissions\r\nThis endpoint requires any one of the [predefined roles](https:\/\/help.vtex.com\/en\/tutorial\/predefined-roles--jGDurZKJHvHJS13LnO7Dy).",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/Content-Type"
+                    },
+                    {
+                        "$ref": "#/components/parameters/Accept"
+                    },
+                    {
+                        "$ref": "#/components/parameters/AudienceParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "array",
+                                    "items": {
+                                        "type": "string",
+                                        "description": "Price table."
+                                    }
+                                },
+                                "example": [
+                                    "priceTable01"
+                                ]
+                            }
+                        }
+                    },
+                    "404": {
+                        "description": "Not Found",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/responseError"
+                                },
+                                "example": {
+                                    "error": "Mapping not found for the provided audienceId."
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "put": {
+                "tags": [
+                    "Price Table Mapper"
+                ],
+                "summary": "Set price table mapping",
+                "description": "Sets the price table mapping for the provided audience.\r\n\r\n## Permissions\r\nThis endpoint requires any one of the [predefined roles](https:\/\/help.vtex.com\/en\/tutorial\/predefined-roles--jGDurZKJHvHJS13LnO7Dy).",
```

Comment:
Please follow [these guidelines](https://www.notion.so/vtexhandbook/Endpoint-permissions-e359ebdc14ac40988095c9936f9db6d5) to write the Permissions section. Which License Manager resources are required to use this endpoint? Which predefined roles work?

It's highly unlikely that "This endpoint requires any one of the predefined roles" to work, considering each predefined role has different resources associated -- unless no resources are necessary, in which case you would go with one of [these templates](https://www.notion.so/vtexhandbook/Endpoint-permissions-e359ebdc14ac40988095c9936f9db6d5?pvs=4#ae3674ee7e974acfa927976d3d522e08), depending on authentication.

I recommend testing or confirming with the team before editing this.

Reply:
Based on the information provided by the developer Conrado, it has been confirmed that there are currently no resources associated with these routes. Therefore, if a user has access to the store, they automatically have access to these endpoints. Consequently, any of the predefined roles enable the use of the endpoint. The team plans to implement specific resource usage in the future, and they will notify me to update the documentation accordingly. Thus, the change made is correct and aligns with the current system configuration.